compiler/luci/lang/include/luci/IR/ProfilingData.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/IR/CircleNode.h"
+
+#include <loco.h>
+
+#include <vector>
+#include <set>
+
+namespace luci
+{
+
+/**
+ * @brief Set annotation for circle node id
+ */
+class CircleNodeID final : public loco::NodeAnnotation
+{
+public:
+  CircleNodeID() = delete;
+
+  CircleNodeID(uint32_t node_id) : _node_id{node_id}
+  {
+    // Do nothing
+  }
+
+public:
+  uint32_t node_id(void) const { return _node_id; }
+  void node_id(uint32_t id) { _node_id = id; } // Do we need this?
```

Comment:
@llFreetimell I think we can remove this setter. Just comment for letting them know that node id shouldn't be changed.

Reply:
For ONE, this setter may not be needed.
But I am not sure that backend does not need this :(

Reply:
I think backend will not need this as of now

Reply:
Maybe just new annotation would be able to applied. I'll follow your choice.

Reply:
If this setter is not used at last, I will remove then :)